File: CoWorking.Contracts/DTO/BookingDTO/BookingInfoDTO.cs
using CoWorking.Contracts.Data.Entities.CommentEntity;
using CoWorking.Contracts.DTO.CommentDTO;

namespace CoWorking.Contracts.DTO.BookingDTO
{
    public class BookingInfoDTO
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public DateTime? DateStart { get; set; }

        public DateTime? DateEnd { get; set; }

        public List<CommentInfoDTO> Comments { get; set; }

        public UserBookingInfoDTO User { get; set; }
    }
}

File: CoWorking.Web/ClientApp/src/app/core/models/booking/BookingInfoDTO.ts
import { CommentInfoDTO } from "../comment/CommentInfoDTO";
import { UserBookingInfoDTO } from "../user/UserBookingInfoDTO";
export class BookingInfoDTO {
    id: number;
    name: string;
    description: string;
    dateStart: Date;
    dateEnd: Date;
    user: UserBookingInfoDTO;
    comments: CommentInfoDTO[];
}
